Odoo 15.0 maintenance

borna-ng opened the pull request:
- Updated base image, Node.js, Python, and Poetry dependencies
- Optimized Dockerfile: combined layers, removed cache, improved reproducibility
- Removed obsolete msfonts.sh script

borna-ng committed:
CPT-806 Update the base Ubuntu image tag

borna-ng committed:
CPT-806 Bump the Node.js version

borna-ng committed:
CPT-806 Update the pyenv version

borna-ng committed:
CPT-806 Update the dependencies with Poetry

borna-ng committed:
CPT-806 Refactor the installation of wkhtmltopdf in Dockerfile

jhelfferich reviewed (dismissed):
Thank you very much for this work :+1:

borna-ng committed:
CPT-806 Remove msfonts.sh

PowerPoint Viewer was retired on April 30, 2018. It's no longer
available for download.

borna-ng committed:
CPT-806 Remove Poetry cache after installing dependencies

borna-ng committed:
CPT-806 Use single COPY for pyproject.toml and poetry.lock

borna-ng committed:
CPT-806 Optimize APT packages installation and make it reproducible

jhelfferich reviewed (approved):
Getting even better 🙂
